docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Google Symptoms
+parent: Data Sources and Signals
+grand_parent: COVIDcast API
+---
+
+# Google Symptoms
+{: .no_toc}
+
+* **Source name:** `google-symptoms`
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
+
+This data source is based on [COVID-19 Search Trends symptoms dataset](https://github.com/google-research/open-covid-19-data/tree/master/data/exports/search_trends_symptoms_dataset). Using this search data, we estimate the volume of searches mapped to symptoms related 
+to COVID-19 such as _anosmia_(lack of smell) and _ageusia_(lack of taste). The resulting daily dataset for each region 
+showing the relative frequency of searches for each symptom.  This signal is measured 
+in arbitrary units that are normalized for population and for the most popular symptom 
+search term within a geographic region. Thus, values are not comparable between 
+geographic regions. Larger numbers represent higher numbers of symptom-related 
```

Comment:
Is it true that values are not comparable between geographic regions, and you can only use time series in one region? I thought the population normalization was supposed to make them comparable somehow.

If they're not comparable, that's a pretty big caveat, and probably deserves its own paragraph with a bold **Warning** label.

Reply:
Yes, they are not comparable across regions since they are scaled by regional specific maximum value at a certain time point.